src/bootstrap/builder.rs

```diff
@@ -330,7 +330,7 @@ impl<'a> ShouldRun<'a> {
 pub enum Kind {
     Build,
     Check,
-    Clippy,
+    Clippy { fix: bool },
```

Comment:
This seems wrong -- nothing else puts arguments here.

Reply:
Changing this requires putting `--fix -Zunstable-options` somewhere other than `args()`. Do you have a suggestion where it should go? This needs to be passed to `clippy-driver` itself so it can't be in `fn cargo()`, maybe I could change args() to take a full `Builder`?

Reply:
Ok, taking a full `Builder` worked fine.